src/test/java/com/google/step/snippet/external/W3SchoolsClientTest.java

```diff
@@ -1,10 +1,9 @@
-package com.google.step.snippet;
+package com.google.step.snippet.external;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
 import com.google.step.snippet.data.Card;
-import com.google.step.snippet.external.W3SchoolsClient;
```

Comment:
Is removing this intentional?

Reply:
Yeah, it's no longer needed now that the package path is fixed